python/kat/harness.py

```diff
@@ -1429,7 +1430,7 @@ def _setup_k8s(self, selected):
                     try:
                         for o in load(n.path, cfg, Tag.MAPPING):
                             parent_config.merge(o)
-                    except YAMLScanError as e:
+                    except (YAMLScanError, YAMLParseError) as e:
```

Comment:
For my understanding:

Does python figure out which type when passing into `Exception` so it can pass friendly messages or is it that since we were only testing for `YAMLScanError` the `YAMLParseError` would get ignored?

Reply:
If the exception doesn't match the types listed, then it gets raised as-is; if it does match then we intercept it and turn it in to `Exception("Parse Error: %s, input text:\n%s" % (e, cfg))` and this is useful because "syntax error on line 40" isn't helpful if you can't actually see the YAML that it's talking about.  I suppose there's an argument to be made that we should just always do that and say `except e:` instead of only doing it for certain types.

Reply:
ok, that is what I thought but wanted to make sure. I'm not familiar enough with the various errors that could be thrown from load but this is probably sufficient to get by for now.